### DISCUSSION TITLE
Return value of moon.phase is not unique!

### DISCUSSION THREAD

**Discussion by Lebostein** [brandon-rhodes/pyephem#281] [General]
Any return value of moon.phase represents 2 completely different states: the waxing or waning moon. Why this function don't return a unmistakable value? Or is there another return parameter that tells me whether, for example, 0.5 means a waxing or waning half moon? Thanks for any tips!

**Comment by brandon-rhodes** [comment]
If you want to know not how illuminated the Moon is (0%–99%), but how far around the sky it is from the Sun, then you can subtract their ecliptic longitudes and get a number of degrees 0°–359.9° where 0° is the moment of New Moon. You can find some example code here:

https://stackoverflow.com/a/26727658/85360
